app/models/custom/user.rb

```diff
@@ -0,0 +1,5 @@
+require_dependency Rails.root.join("app", "models", "user").to_s
+
+class User
+  devise :lockable if Rails.application.config.devise_lockable
```

Comment:
We can add this line to the `app/models/user.rb` file directly. It seems to work fine!

```
  devise :database_authenticatable, :registerable, :confirmable, :recoverable, :rememberable,
         :trackable, :validatable, :omniauthable, :password_expirable, :secure_validatable,
         authentication_keys: [:login]
  devise :lockable if Rails.application.config.devise_lockable
```